api_app/script_analyzers/observable_analyzers/urlscan.py

```diff
@@ -0,0 +1,81 @@
+import requests
+import time
+import json
+from api_app.exceptions import AnalyzerRunException
+from api_app.script_analyzers import classes
+from intel_owl import secrets
+
+
+class UrlScan(classes.ObservableAnalyzer):
+    base_url: str = "https://urlscan.io/"
+
+    def set_config(self, additional_config_params):
+        self.analysis_type = additional_config_params.get("urlscan_analysis", "search")
+        api_key_name = additional_config_params.get("api_key_name", "URLSCAN_KEY")
+        self.visibility = additional_config_params.get("visibility", "public")
+        self.__api_key = secrets.get_secret(api_key_name)
+
+    def __submit_query(self):
+        data = {"url": self.observable_name, "visibility": self.visibility}
+        uri = "api/v1/scan/"
+        try:
+            response = requests.post(
+                self.base_url + uri, headers=self.headers, data=json.dumps(data)
```

Comment:
It is better to use the `json` parameter instead of `data`. Check the docs in `requests`

Reply:
Change `data=json.dumps(data)` to `json=data`. Thanks!